PinpointKit/PinpointKit/Sources/BarButtonItem.swift

```diff
@@ -14,12 +14,16 @@ extension UIBarButtonItem {
      Convenience initializer for creating a “Done” `UIBarButtonItem` with a specified target, action, and font.
      
      - parameter target: The bar button item’s target.
+     - parameter title:  The bar button item’s title.
```

Comment:
Could you please document the functionality of passing `nil` or using the default parameter value (i.e. that it uses "Done")?